docs/news.rst

```diff
@@ -34,7 +34,7 @@ See :ref:`option_input_output_file` in :ref:`ctags(1) <ctags(1)>`.
 People may use their own native language in source code comments (or
 sometimes in identifiers) and in such cases encoding may become an issue.
 Nowadays UTF-8 is the most widely used encoding, but some source codes
-still use legacy encodings like latin1, cp932 and so on. These options
+still use legacy encodings like latin1, shift_jis and so on. These options
```

Comment:
I don't think this needs to be changed.

Reply:
If this not changed, `musl-libc` user also will be confused when they find `cp932` is not work.

Reply:
As a Japanese user, many Windows source codes were written in cp932 not in shift_jis, using shift_jis as an example confuses many Japanese Windows users.

Reply:
`cp932` coding is motioned in our docs, but it can not work when os use `musl-libc`, and our docs don't provide a solution, I think these people who really coming to find answers will be confused.

Reply:
The example `cp932` is for Japanese Windows users (as the iconv feature was originally created for them), and win32 u-ctags uses GNU iconv not musl.

Reply:
OK. I will switch `docs/news.rst` back to Origin.